_posts/2022-02-02-#748.md

```diff
@@ -1,8 +1,9 @@
 ---
 layout: pr
 date: 2022-02-02
-title: "Add usage examples"
+title: "Libsecp256k1: Add usage examples"
```

Comment:
surely this won't be the last one? :scream:
```suggestion
title: "Libsecp256k1 #748: Add usage examples"
```

Reply:
DONE